modules/network-observability-quickfilter.adoc

```diff
@@ -12,6 +12,7 @@ You can use *Query Options* to optimize the search results, as listed below:
 
 ** *Log Type*: The available options *Conversation* and *Flows* provide the ability to query flows by log type, such as flow log, new conversation, completed conversation, and a heartbeat, which is a periodic record with updates for long conversations. A conversation is an aggregation of flows between the same peers.
 ** *Match filters*: You can determine the relation between different filter parameters selected in the advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.
+** *Datasource*: You can choose the datasource to use for queries: *Loki*, *Prometheus*, or *Auto*. Notable performance improvements can be realized when using Prometheus as a datasource compared to Loki.
```

Comment:
@jotak are there any other usage things to note about datasource query options? Did I capture everything accurately? Also is Auto default?

Reply:
Yes, auto is the default. Perhaps you can add:

[...] Notable performance improvements can be realized when using Prometheus as a datasource compared to Loki, but supports a limited set of filters and aggregations. *Auto* mode will use Prometheus on supported queries, or fall back to Loki otherwise.